### DISCUSSION TITLE
posts/installing-newer-python-amazon-ec2/

### DISCUSSION THREAD

**Discussion by giscus** [hkamran80/articles#28] [Comments]
# posts/installing-newer-python-amazon-ec2/

EC2 ships with Python 3.7.11, but some might need a newer version

https://hkamran.com/posts/installing-newer-python-amazon-ec2/

**Comment by DrB-S** [comment]
Will yum and other applications dependent upon python2 work with python3.11 and openssl11-devel on Amazon EC2?

**Reply by hkamran80** [reply]
I have `python3.11` installed alongside the preinstalled binaries that come with Amazon Linux 2. `yum` and other commands continue to work normally for me.

**Comment by shaunster666** [comment]
Very useful, many thanks

**Reply by hkamran80** [reply]
You're welcome!
